### PR TITLE
No longer need to include rubygems or bundler software dep

### PR DESCRIPTION
**Pull request by tyler-ball**
## Description
Recent versions of Ruby have included Rubygems and Bundler, which they
did not always do. We also recently fixed some issues in Appbundler so
these software dependencies are no longer needed.

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New content (non-breaking change)
- [ ] Breaking change (a content change which would break existing functionality or processes)

## Checklist:

- [x] I have read the **CONTRIBUTING** document.

### CODE REVIEW

**Comment by tyler-ball** [comment]
I kicked off an ad-hoc build for this. If it builds correctly, can someone install the package and manually verify it has rubygems and bundler in the gem list? It shouldn't be able to build without those but I think it is worth doing some due diligence

File: omnibus_overrides.rb [hunk 1]
```diff
@@ -4,8 +4,6 @@
 #
 # NOTE: You MUST update omnibus-software when adding new versions of
 # software here: bundle exec rake dependencies:update_omnibus_gemfile_lock
```
Comment (james-stocks) [inline omnibus_overrides.rb]: It may be helpful to insert a blank line to keep the comment about train separate from the comment block above

**Comment by james-stocks** [comment]
Passing ad-hoc build: https://buildkite.com/chef/inspec-inspec-master-omnibus-adhoc/builds/129#35c0e285-884a-4206-b440-3813f9df4a8a

**Comment by Schwad** [comment]
Nice catch @tyler-ball - thanks!

**Comment by james-stocks** [comment]
@tyler-ball - I've installed http://artifactory.chef.co/simple/omnibus-unstable-local/com/getchef/inspec/4.18.113+20200508234600/windows/10/ and it comes with bundler 1.17.2 as a gem and C:\opscode\inspec\embedded\lib\ruby\2.6.0\rubygems is included in the install

A quick test of `inspec shell` shows the install as working. Anything else to check?

File: omnibus_overrides.rb (hunk 1 above)
Comment (btm) [reply omnibus_overrides.rb]: I added a commit to this PR removing that comment altogether. It's actually not relevant in this project.

**Review by btm — APPROVED** [review]

**Comment by btm** [comment]
@james-stocks we've been making this change in all of our projects recently, so confidence is high this just works and I think you've adequately checked everything.

**Review by james-stocks — APPROVED** [review]

File: omnibus_overrides.rb [hunk 2]
```diff
@@ -1,11 +1,6 @@
 # THIS IS NOW HAND MANAGED, JUST EDIT THE THING
-# .travis.yml and appveyor.yml consume this,
-# try to keep it machine-parsable.
+# .travis.yml and appveyor.yml consume this, try to keep it machine-parsable.
```
Comment (tas50) [inline omnibus_overrides.rb]: This comment is no longer valid. We used it previously to install bundler / rubygems

Comment (lamont-granquist) [reply omnibus_overrides.rb]: technically its not a bad thing to keep it machine-parseable though.

Comment (tas50) [reply omnibus_overrides.rb]: I meant just the bit about travis / appveyor

Comment (btm) [reply omnibus_overrides.rb]: I removed the whole line. I was going to say "Please try to keep it machine-parsable" but then the obvious question was "for what? how do I know if the machine can parse this or not" and it isn't really doing anyone favors.

**Pull request merged**